alembic/versions/2d0f5c131dba_added_rate_limit_table.py

```diff
@@ -0,0 +1,37 @@
+"""added Rate Limits table
+
+Revision ID: 2d0f5c131dba
+Revises: eca40a718b7a
+Create Date: 2024-07-24 23:25:59.269937
+
+"""
```

Comment:
nice idea,

this is not needed now

changes to the model and migrations are not allowed

make necessary changes

Reply:
Hi, thanks for the prompt review.

What I did there is what I thought is feasible. The task in the ticket #104 requires me to implement Rate limiting to ensure user does not send too many requests to the endpoint.

In order to achieve the desired result, I can either use redis(this way I'd be introducing new technology to the system) or database.
I opt in for database which is easier and quicker but it will require a Table in the database and a new model.

What do you suggest I do in order to resolve this sir?

Reply:
omit the rate limiting for now, we didn't add that in the db design.

moving forward we could take that into consideration.